src/templates/app/user/set_currency.jsx

```diff
@@ -30,12 +30,16 @@ const SetCurrency = () => (
                     <img src={it.url_for('images/pages/set_currency/account-created.svg')} />
                     <div className='gr-padding-10' />
                     <h1>{it.L('Congratulations!')}</h1>
-                    <p>{it.L('You have successfully created your account!')}</p>
+                    <p>
+                        {it.L('You have successfully created your [_1]account!', `<span data-show="iom">${it.L('gaming')}&nbsp;</span>`)}
+                        <br/>
+                        <span className='invisible has_currency'>{it.L('Please [_1]authenticate[_2] your account before you make a deposit.', `<a href="${it.url_for('user/authenticate')}">`, '</a>')}</span>
+                    </p>
                 </div>
 
                 <SeparatorLine show_mobile className='gr-parent gr-padding-10' invisible />
 
-                <div className='invisible' id='has_currency'>
+                <div className='invisible has_currency'>
```

Comment:
does this need to be a class? can it remain as id?

Reply:
yes, `has_currency` is used in line 36 as well